pxtblocks/fields/field_textdropdown.ts

```diff
@@ -140,11 +140,11 @@ export class BaseFieldTextDropdown extends Blockly.FieldTextInput {
         for (let i = 0; i < options.length; i++) {
             const [label, value] = options[i];
             const content = (() => {
-                if (typeof label === 'object') {
+                if (isImageProperties(label)) {
```

Comment:
Align with Richard's approach. Worried this is stricter re alt.

Reply:
Checked and alt is not optional. Shared the utility between the two scenarios which is as per the Blockly code.